### DISCUSSION TITLE
More than 4 shutters for ESP32 as a compile option

### DISCUSSION THREAD

**Discussion by SilentBob83** [arendst/Tasmota#13711] [Q&A] [answered]
hello i would like to ask if it could be made possible to have more than 4 shutters maybe as a compile option for ESP32 ?
I would need to control 6 shutters and do not really like to install a second hardware....

I have already asked about that topic here:

https://github.com/arendst/Tasmota/issues/8921

best regards

Andreas

**Comment by Jason2866** [comment, marked as answer]
A compile option is not enough. Max. 4 shutters is a design limit. It needs a redesign (code changes) of Tasmota ~~and shutter code~~ to make this possible.
Edit see comment https://github.com/arendst/Tasmota/issues/8921#issuecomment-966858389

**Comment by SilentBob83** [comment]
Hello are there any updates on this topic ?

**Reply by Jason2866** [reply]
No.

**Comment by greg741** [comment]
Hi,

I have build Tasmota 9.5.0.9 for ESP32 with support up to 10 shutters.
The image has support for PCF8574.
I you want to test it, please download [firmware ESP32.](https://github.com/greg741/Tasmota-9.5.0.9/tree/main/firmware)

**Comment by SilentBob83** [comment]
Hallo... nice and thanks for the Info. I am currently in rehabilitation. I will be back in 2 Werkstatt. Than i would Luke to test it.Best regardsAndreas Am 21.07.2022 22:03 schrieb Grzegorz ***@***.***>:
Hi,
I have build Tasmota 9.5.0.9 for ESP32 with support up to 10 shutters.
The image has support for PCF8574.
If you want to test it, let me know.
